components/ratgdo/ratgdo.cpp

```diff
@@ -21,7 +21,6 @@ namespace esphome {
 namespace ratgdo {
 
     static const char* const TAG = "ratgdo";
-    static const int STARTUP_DELAY = 2000; // delay before enabling interrupts
     static const uint8_t MAX_CODES_WITHOUT_FLASH_WRITE = 3;
```

Comment:
Is this unused now ?

Reply:
Seems like it's been unused for a while, I noticed it when checking for `delay` calls in the code and didn't find any place where it was used.